docs/howto/installing-local-extensions.md

```diff
@@ -92,12 +92,19 @@ Step 4
 
 Bolt might access your public assets (javascripts, stylesheets, images, etc.).
 For local extension, copy your assets manually to :
-`/{public_root}/extensions/local/{author_name}/{extension_name}/`
+`{public_root}/extensions/local/{author_name}/{extension_name}/`
+
+or just create symlink ( not guaranteed )
```

Comment:
Not supported means we won't add them to the documentation, as that implys we do.